draft-ietf-sidrops-rpki-prefixlist.xml

```diff
@@ -270,12 +270,12 @@
       <name>Security Considerations</name>
       <t>
         Relying Parties are warned that the data in a Signed Prefix List is self-asserted by the AS holder.
-        There is no implied authority from any IP prefix holder that grants the AS permission to originate a route for any prefixes.
+        There is no implied authority in a Signed Prefix List that any IP prefix holder has granted the AS permission to originate a route for any of the listed prefixes.
         Such an authority is separately conveyed in the RPKI as a ROA.
       </t>
       <t>
-        While a one-time-use EE certificate must only be used to generate and sign a single Signed Prefix List object, CAs are not technically restricted from generating and signing multiple different Signed Prefix List objects with a single key pair.
-        Any Signed Prefix List objects sharing the same EE certificate cannot be revoked individually.
+        While a one-time-use EE certificate are only be used to generate and sign a single Signed Prefix List object, CAs are not technically restricted from generating and signing multiple different Signed Prefix List objects with a single key pair.
```

Comment:
This first sentence does not flow very well;

```suggestion
        While the key pairs of one-time-use EE certificates are only used to generate and sign a single Signed Prefix List object, CAs are not technically restricted from generating and signing multiple different Signed Prefix List objects with a single key pair.
```

Reply:
Thanks, I worded it slightly different: https://github.com/job/draft-sidrops-rpki-prefixlist/commit/9732c6daf43fc2e0c8bc1ecabd8533613b5eaf7a